e2e/app/page/cohort-search-page.ts

```diff
@@ -94,28 +108,10 @@ export default class CohortCriteriaModal extends Modal {
     const link = await this.waitForPhysicalMeasurementCriteriaLink(criteriaName);
     await link.click();
 
-    const selectMenu = await SelectMenu.findByName(this.page, {ancestorLevel: 2}, this);
-    await selectMenu.clickMenuItem(filterSign);
-
-    const numberField = await this.page.waitForXPath(`${this.xpath}//input[@type="number"]`, {visible: true});
-    await numberField.type(String(filterValue));
-
-    await this.clickButton(LinkText.Calculate);
-    const participantResult = await this.waitForParticipantResult();
+    const helpSidebar = await new HelpSidebar(this.page);
```

Comment:
nit: remove `await` in class constructor call.

Reply:
Done.